test/test_admin_integration.py

```diff
@@ -80,3 +81,57 @@ def test_create_describe_delete_acls(kafka_admin_client):
 
     assert error is NoError
     assert len(acls) == 0
+
+
+@pytest.mark.skipif(env_kafka_version() < (0, 11), reason="Describe config features require broker >=0.11")
```

Comment:
As a comment, since all these tests require a version, you could probably have organized them into a pytest class (which is primarily for organization, doesn't actually do most typical class/object style stuff) and then applied the decorator to the class and it would be applied to all the tests within that class. I'm not worried about it, more just an FYI if you ever work on stuff like this in the future...

Reply:
Thanks for the heads up - i'll keep that in mind for future work!